nautobot/extras/models/models.py

```diff
@@ -769,28 +771,38 @@ def check_for_conflicts(cls, instance, content_types, payload_url, type_create,
         conflicts = {}
         webhook_error_msg = "A webhook already exists for {action} on {content_type} to URL {url}"
 
-        for content_type in content_types:
-            webhooks = cls.objects.filter(content_types__in=[content_type], payload_url=payload_url)
-            if instance and instance.present_in_database:
-                webhooks = webhooks.exclude(pk=instance.pk)
-
-            existing_type_create = webhooks.filter(type_create=type_create).exists() if type_create else False
-            existing_type_update = webhooks.filter(type_update=type_update).exists() if type_update else False
-            existing_type_delete = webhooks.filter(type_delete=type_delete).exists() if type_delete else False
-
-            if existing_type_create:
-                conflicts.setdefault("type_create", []).append(
-                    webhook_error_msg.format(content_type=content_type, action="create", url=payload_url),
-                )
-
-            if existing_type_update:
-                conflicts.setdefault("type_update", []).append(
-                    webhook_error_msg.format(content_type=content_type, action="update", url=payload_url),
-                )
-
-            if existing_type_delete:
-                conflicts.setdefault("type_delete", []).append(
-                    webhook_error_msg.format(content_type=content_type, action="delete", url=payload_url),
-                )
+        if content_types is None and instance is not None:
```

Comment:
I still don't see why the `if content_types is None` gates the logic for `payload_url`, `type_create`, `type_update`, `type_delete`. What am I missing?